packages/rich-text/src/component/index.native.js

```diff
@@ -271,6 +274,13 @@ export class RichText extends Component {
 			.replace( closingTagRegexp, '' );
 	}
 
+	// Fix for crash https://github.com/wordpress-mobile/gutenberg-mobile/issues/2991
+	convertFontSizeFromString( fontSize ) {
+		return fontSize && fontSize.endsWith( 'px' )
```

Comment:
Not sure, I wonder if there could be more cases of `value+unit` other than `px`. Like, `pt` or `em`.

Should be fine to fix only for px now, but feels like we should take the opportunity to improve detecting this issue. The error message we encounter is [java.lang.String cannot be cast to java.lang.Double](java.lang.String cannot be cast to java.lang.Double) on Android, and it will be that again if a different unit is encountered, making it difficult for us to know if we really fixed the crash or not. Any ideas how to signal that something else than `px` was found?

I think that we don't have a good way to add breadcrumbs to the Sentry logs, but maybe you have some idea Cameron?